go.mod

```diff
@@ -0,0 +1,64 @@
+module github.com/PaloAltoNetworks/terraform-google-vmseries-modules
+
+go 1.20
+
+require (
+	github.com/PaloAltoNetworks/terraform-modules-vmseries-tests-skeleton v1.0.7-0.20230921084834-413e7c888833
```

Comment:
This needs to changed after removing branch and releasing new version of `terraform-modules-vmseries-tests-skeleton `

Reply:
Of course - it was left like that just to be able to test the access via OIDC using the pull request trigger !